src/core/maprenderer/qgsmaprendererjob.h

```diff
@@ -440,7 +459,7 @@ class CORE_EXPORT QgsMapRendererJob : public QObject
      * \note not available in Python bindings
      * \since QGIS 3.12
      */
-    static void composeSecondPass( LayerRenderJobs &secondPassJobs, LabelRenderJob &labelJob ) SIP_SKIP;
+    static void composeSecondPass( LayerRenderJobs &secondPassJobs, LabelRenderJob &labelJob, bool forceVector, bool hasClipping, QPainterPath clipPath ) SIP_SKIP;
```

Comment:
Should QPainterPath be received by const reference here ?

Reply:
Yes I think it can.